docker-compose.yml

```diff
@@ -12,13 +12,13 @@ services:
 
   db_registry:
     hostname: db_registry
-    image: postgres:13
+    image: postgres:12.6
```

Comment:
please don't modify default config, this is deployment specific and can be overwritten